language/de/cli.php

```diff
@@ -160,6 +162,10 @@
 
 	'CLI_USER_ADD_SUCCESS'		=> 'Benutzer %s erfolgreich angelegt.',
 	'CLI_USER_DELETE_CONFIRM'	=> 'Soll „%s“ wirklich gelöscht werden? [y/N]',
+	'CLI_USER_DELETE_ID_CONFIRM'	=> 'Bist du sicher, dass du die Benutzer mit den IDs „%s“ löschen möchtest? [y/N]',
```

Comment:
y/N oder j/N?

Reply:
`j/N` wird wohl nicht gehen, wenn das in Symfony Code so aussieht:

**vendor\symfony\console\Question\ConfirmationQuestion.php**
`public function __construct($question, $default = true, $trueAnswerRegex = '/^y/i')`

Reply:
Irgendwie müsste sowas eigentlich mit eindeutschbar sein, wenn man schon das Kommandozeileninterface übersetzt.

Reply:
Lässt sich also nicht ändern --> Resolved.